api/envoy/admin/v2alpha/clusters.proto

```diff
@@ -28,8 +28,8 @@ message ClusterStatus {
   // Denotes whether this cluster was added via API or configured statically.
   bool added_via_api = 2;
 
-  // The success rate threshold used in the last interval. The threshold is used to eject hosts
-  // based on their success rate. See
+  // The success rate threshold used in the last interval for externally generated failures.
```

Comment:
Please add a release note for all the changes in this PR.

Reply:
Also, should this doc reference the `split_external_local_origin_errors` setting so it's clear what this field sets depending on the setting? Feel free to link back to the arch docs if that would be easier.

Reply:
Release note added.